code/_helpers/global_lists_ch.dm

```diff
@@ -84,101 +84,106 @@ var/global/list/vr_mob_tf_options = list(
+	"Sheep" = /mob/living/simple_mob/vore/sheep,
+	"Slug" = /mob/living/simple_mob/vore/slug,
+	"Snake" = /mob/living/simple_mob/animal/passive/snake,
+	"Sobaka" = /mob/living/carbon/human/sharkm,
 	"Solar grub" = /mob/living/simple_mob/vore/solargrub,
-	"Pitcher plant" = /mob/living/simple_mob/vore/pitcher_plant,
-	"Red gummy kobold" = /mob/living/simple_mob/vore/candy/redcabold,
-	"Blue gummy kobold" = /mob/living/simple_mob/vore/candy/bluecabold,
-	"Yellow gummy kobold" = /mob/living/simple_mob/vore/candy/yellowcabold,
-	"Marshmellow serpent" = /mob/living/simple_mob/vore/candy/marshmellowserpent,
-	"Riftwalker" = /mob/living/simple_mob/vore/demon,
+	"Space bear" = /mob/living/simple_mob/animal/space/bear,
+	"Space bumblebee" = /mob/living/simple_mob/vore/bee,
+	"Space carp" = /mob/living/simple_mob/animal/space/carp,
+	"Space ghost" = /mob/living/simple_mob/vore/alienanimals/spooky_ghost,
+	"Space jelly fish" = /mob/living/simple_mob/vore/alienanimals/space_jellyfish,
+	"Space shark" = /mob/living/simple_mob/animal/space/shark,
+	"Sparra" = /mob/living/carbon/human/sparram,
+	"Squirrel" = /mob/living/simple_mob/vore/squirrel,
+	"Star treader" = /mob/living/simple_mob/vore/alienanimals/startreader,
+	"Stok" = /mob/living/carbon/human/stok,
+	"Succubus" = /mob/living/simple_mob/vore/succubus,
+	"Super metroid" = /mob/living/simple_mob/metroid/juvenile/super,
+	"Synx" = /mob/living/simple_mob/animal/synx,
+	"Teppi" = /mob/living/simple_mob/vore/alienanimals/teppi,
+	"Vampire" = /mob/living/simple_mob/vore/vampire/countess_nude,
+	"Voracious lizard" = /mob/living/simple_mob/vore/aggressive/dino,
 	"Wendigo" = /mob/living/simple_mob/vore/demon/wendigo,
-	"Shadekin" = /mob/living/simple_mob/shadekin,
-	"Catgirl" = /mob/living/simple_mob/vore/catgirl,
+	"Weretiger" = /mob/living/simple_mob/vore/weretiger,
 	"Wolfgirl" = /mob/living/simple_mob/vore/wolfgirl,
 	"Wolftaur" = /mob/living/simple_mob/vore/wolftaur,
-	"Lamia" = /mob/living/simple_mob/vore/lamia,
-	"Corrupt hound" = /mob/living/simple_mob/vore/aggressive/corrupthound,
-	"Corrupt corrupt hound" = /mob/living/simple_mob/vore/aggressive/corrupthound/prettyboi,
-	"SWOOPIE XL" = /mob/living/simple_mob/vore/aggressive/corrupthound/swoopie,
-	"Cultist Teshari" = /mob/living/simple_mob/humanoid/cultist/tesh,
-	"Burning Mage" = /mob/living/simple_mob/humanoid/cultist/human/bloodjaunt/fireball,
-	"Converted" = /mob/living/simple_mob/humanoid/cultist/noodle,
-	"Cultist Teshari Mage" = /mob/living/simple_mob/humanoid/cultist/castertesh,
-	"Monkey" = /mob/living/carbon/human/monkey,
 	"Wolpin" = /mob/living/carbon/human/wolpin,
-	"Sparra" = /mob/living/carbon/human/sparram,
-	"Saru" = /mob/living/carbon/human/sergallingm,
-	"Sobaka" = /mob/living/carbon/human/sharkm,
-	"Farwa" = /mob/living/carbon/human/farwa,
-	"Neaera" = /mob/living/carbon/human/neaera,
-	"Stok" = /mob/living/carbon/human/stok,
-	//"Gryphon" = /mob/living/simple_mob/vore/gryphon // Disabled until tested
```

Comment:
This has been tested and works pretty much as intended. The only way it's going to get tested more is if you give it to players to play with.